install.sh

```diff
@@ -187,14 +186,27 @@ check_deps() {
 node(){
   # clean up 
   rm -rf node 
+  rm -f SHASUMS256.txt.asc
   rm -rf node-$NODE_VERSION*
-  
+
   echo :Installing Node $NODE_VERSION
-  
+
+  gpg --list-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D 2>&1 >/dev/null || gpg --keyserver pool.sks-keyservers.net --recv-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D
```

Comment:
GPG is much more secure. The line of trust for GPG is:
Trusting the developer -> however you got the source (via SSH being more secure than https, https making this the same as the next case)
The line of trust for HTTPS is:
Trusting the developer -> however you got the source -> everybody who can sign certificates trusted by your browser (including terribly insecure organizations and many nation-states).

I'm trying to make sure that as few cases of the second scenario exist in my life as possible.

That said, most people clone stuff from GitHub via HTTPS, so they're probably not going to derive much benefit from this.